cmd/create.go

```diff
@@ -146,11 +156,42 @@ A full list of component types that can be deployed is available using: 'odo cat
 			os.Exit(1)
 		}
 
+		// If min-memory, max-memory and memory are passed, memory will be ignored as the other 2 have greater precedence.
+		// Emit a message indicating the same
+		if memoryMin != "" && memoryMax != "" && memory != "" {
+			fmt.Println("Flag `memory` will be ignored as `min-memory` and `max-memory` have been passed ")
```

Comment:
This seems weird compared to others..

It should be `--memory will be ignored as --min-memory and --max-memory has been passed`

Reply:
Ack